external/nmea/parse.c

```diff
@@ -390,7 +390,7 @@ int nmea_parse_GPGSV( const char *buff, int buff_sz, nmeaGPGSV *pack )
     return 0;
   }
 
-  if ( type != 'P' && type != 'N' )
+  if (type != 'P' && type != 'N' && type != 'L' && type != 'A' && type != 'B' && type != 'Q')
```

Comment:
```suggestion
  if ( type != 'P' && type != 'N' && type != 'L' && type != 'A' && type != 'B' && type != 'Q' )
```

Reply:
ok.  
This is my first time making a pull request.  
I am not a programmer!  
I have to do something?